docs/src/pages/careers/react-tech-lead-core.md

```diff
@@ -0,0 +1,122 @@
+  It's fun, rewarding work, but it can be very challenging.
+- **We move quickly, but don't sacrifice quality**.
+  We ship early, often, and quickly. You may not be initially comfortable with the cadence with which we ship high-quality features and improvements to end-users. By doing so, we sacrifice on solving each problem 100% in exchange for fast feedback. Solving 50-70% of the issue with quality should be enough for any given iteration. Our users quickly tell us when we haven't pushed a solution far enough.
+- **MUI is a large codebase**. You may bang your head against the wall at times, and then write tests to assist future you 😌.
+  The work you will be doing is somewhat unique and idiosyncratic. You probably have not had a similar role before.
+- **In open-source, you're faced with a nonstop stream of bug reports and support requests**. That means you need to develop an intuition for when to ignore something, and when to dig in further.
+
+## About you
+
+We're looking for someone with strong front-end skills. More important than specific technical skills though is that you're a strong problem solver who loves to learn. Details matter to you.
+
+### Skills you should have
+
+- **Expertise in the modern JavaScript ecosystem**.
+  MUI is built on the shoulders of giants, making use of technologies such as ES2021, TypeScript, Node.js, React, Next.js, webpack, and Babel.
+- **A track record of demonstrating an eye for product and solving real-world user problems**. If you have a knack for solving problems at the root cause, shipping beautiful user interfaces and intuitive APIs, we want you on our team.
+- **Experience building and shipping production code in a team setting** with a passion for writing tested, performant, and high-quality code.
+- **Strong written and verbal communication skills**.
+  As part of the team, you'll interface both directly and indirectly with community members and enterprise customers, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
+- **Ability to dive into complex problems**.
+  You should be able to quickly assess, understand, and iterate upon aspects of our codebase.
+- **Ready and willing to ask and answer questions**.
+  If you're comfortable saying you're unsure, asking for help; but equally reaching out to assist others, you'll be an incredible addition to our team. We thrive because of continuous learning. First-time mistakes should be celebrated, not blamed.
+- **Avoid monolithic deliverables**.
+  You scope and stage your work into well-defined milestones to ship.
+- **Past work with front-end infrastructure**.
+  Perhaps you've created your company's design system, written a Babel plugin, or written complex React components.
+  It would be great if you could address this in your cover letter!
+
+### What it would be nice if you had, but isn't required
+
+- **You've contributed to a design system before**.
```

Comment:
I would say that for the Tech Lead role this may even be a requirement, not a "nice to have" thing.

Reply:
Sold, it copy & pasted this too quickly.